scripts/setup-linux.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eufx -o pipefail
+SCRIPTDIR=$(dirname "${BASH_SOURCE[0]}")
+source $SCRIPTDIR/setup-common.sh
+
+function install_hdfs {
```

Comment:
Why is this not in common?

Reply:
The macos version below does not have hadoop installed. Everything else is the same. Hadoop is required for testing. Let's add this hadoop install directly to ubuntu/centos install_hdfs and remove setup-linux.sh